src/resources/audio.js

```diff
@@ -73,7 +74,7 @@ class AudioHandler {
     _isSupported(url) {
         const ext = path.getExtension(url);
 
-        if (toMIME[ext]) {
+        if (supportedExtensions.includes(ext)) {
```

Comment:
IE11 doesn't have `Array.prototype.includes`. Maybe change to `supportExtensions.indexOf(ext) !== -1`

Reply:
Should I also fix it here? This is the only other place where Array.prototype.includes is used outside of tests and examples:
https://github.com/playcanvas/engine/blob/08b1f08622f128d1a6bd9df21b8b48a3cd17b2f3/src/framework/components/anim/system.js#L44
Or you want this in a separate pull?

That it's also used here tells me that nobody ever tests it on IE11 and it might be broken already in other places as well?

Reply:
We do test on IE11 but only the examples browser application. So if none of the examples hit that line, the test plan would pass. I just went to check, but since the last release, a new IE11-incompatible change has been introduced: https://github.com/playcanvas/engine/issues/4287
When that's fixed, I'll check the anim-related examples.
I would leave that occurrence of `includes` out of this PR. Feel free to open a new one though.

Reply:
Ok I removed `supportedExtensions.includes` from my pull only.